src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/IcraEligibility.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <v-container>
+    <div>
+      <Breadcrumb />
+      <h1 class="pb-9">Apply with international certification</h1>
+
+      <ECEHeader title="Is this pathway right for you?" />
+      <p>
+        If you are certified as an Early Childhood Educator (ECE) in another country, you may be eligible to apply for ECE Five Year certification in British
+        Columbia using professional experience you have outside of Canada.
+      </p>
+      <p class="mt-5 mb-8">
+        <b>Note:</b>
+        This application is only for
+        <b>ECE Five Year certification.</b>
+        It does not apply to ECE Assistant of ECE One Year certification.
+      </p>
+      <ECEHeader title="The eligibility process" />
+      <p class="mt-3">1. Submit your required documentation</p>
+      <p class="mt-1">
+        2. We will review your submission and determine if you are eligible to apply for ECE Five Year certification using professional experience you have
+        outside of Canada
+      </p>
+      <p class="mt-1">
+        3. If you are eligible, you may continue with your ECE Five Year application. If you are not eligible, we will contact you with other application
+        options
+      </p>
+
+      <Callout class="mt-10" type="warning" title="Do you have 500 hours of work experience supervised by a Canadian-certified ECE?">
+        If yes, you do not need to confirm your eligibility. Please
+        <router-link class="mt-4" to="/">apply for certification directly here.</router-link>
+      </Callout>
+
+      <h3 class="mt-13">When you are ready, you can begin your submission.</h3>
+
+      <v-btn class="mt-10" id="backToHome" size="large" color="primary" @click="handleBackToHome">
+        <v-icon size="small" icon="mdi-arrow-left" class="mr-2"></v-icon>
+        Back to home
+      </v-btn>
+    </div>
```

Comment:
I think we need that check eligibility button here that will eventually lead to my page? 

<img width="493" height="147" alt="image" src="https://github.com/user-attachments/assets/e42240f4-c1b5-41fa-9e12-e18d793f368d" />

Reply:
p.s. to save on creating another method for this button click, I cheated and just put it with the button 

<v-btn class="mt-6" rounded="lg" color="primary" @click="router.push({ name: '<name of router path here>' })" id="btnContinue">Continue (not implemented)</v-btn>